src/styles/blocks/header.scss

```diff
@@ -0,0 +1,15 @@
+// --text-color: #00acdc;
```

Comment:
The commented-out CSS variable '--text-color' could be removed if it's not being used, to keep the code clean. Alternatively, if it's intended for future use, consider adding a comment explaining its purpose.